<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskSheetEditor</class>
 <widget class="QMainWindow" name="TaskSheetEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>合同编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contractEdit">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>业务人员：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="salesRepresentativeBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>报告用途：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="reportPurposEdit">
        <item>
         <property name="text">
          <string>企业自行监测</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>建设项目环评</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>建设项目验证</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>监督执法</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其它</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>委托单位信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>委托单位名称：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>委托单位地址：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="clientAddrEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>委托单位联系人：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>委托单位联系电话：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="clientContactsPhoneEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MyComboBox" name="clientBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="clientContactsBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>受检单位信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="3">
        <widget class="QLineEdit" name="inspectedPhoneEidt">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>受检单位联系人：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="projectNameEdit"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="inspectedAddrEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>受检单位地址：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MyComboBox" name="inspectedComBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>项目地址：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>受检单位联系电话：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>受检单位名称：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>项目名称：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="projectAddrEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="inspectedContactsBox">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>与委托单位一致</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>客户需求</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>留样处理：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>报告份数：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>是否分包：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>其它要求：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="otherRequirementsEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>报告周期：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>检测方法：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="reportPeriodBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="reportCopiesBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="sampleDisposalBox">
         <item>
          <property name="text">
           <string>由受托方处理</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>退回委托方</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="subpackageBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>不可分包</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>允许分包</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="methodSourseBox">
         <item>
          <property name="text">
           <string>由实验室确认</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>委托方提供</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>检测信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="2" column="0" colspan="4">
        <widget class="MyTableView" name="testInfoTableView"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="commitBtn">
         <property name="text">
          <string>提交评审</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="methodSelectBtn">
         <property name="text">
          <string>进入方法选择</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="exportBtn">
         <property name="text">
          <string>导出EXCEL</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>276</width>
         <height>59</height>
        </rect>
       </property>
       <property name="title">
        <string>检测项目</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>样品类别</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>检测领域</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="sampleTypeBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="testFiledBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>检测类型</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>样品类别</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="testTypeBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="testItemAddBtn">
          <property name="text">
           <string>选择检测参数</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextEdit" name="testItemEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>按执行标准</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>手动输入</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>选择项目</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按历史项目</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="testInofOkBtn">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="standardSelectBtn">
          <property name="text">
           <string>选择执行标准</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>添加委托单位</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>添加受检单位</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>mytableview.h</header>
  </customwidget>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
